src/transformers/trainer.py

```diff
@@ -663,6 +694,11 @@ def _prediction_loop(
         else:
             metrics = {}
         if len(eval_losses) > 0:
-            metrics["loss"] = np.mean(eval_losses)
+            metrics["eval_loss"] = np.mean(eval_losses)
+
+        # Prefix all keys with eval_
+        for key in list(metrics.keys()):
+            if not key.startswith("eval_"):
+                metrics[f"eval_{key}"] = metrics.pop(key)
```

Comment:
@jplu I changed the attribute names of the output of `_prediction_loop()` to ensure that all keys start with `eval_`

This might be something we would want to also do in TF for consistency